README.md

```diff
@@ -36,6 +43,14 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+1. Smart Table: A pad in the center that you put your cup on and alerts the bar/waiter that a cup needs to be refilled. Would also have a screen for drink customization.
+    * Lets people order whatever drink they want as soon as their cup is empty.
+    * Has to wait for drink to be empty to alert that a new drink is needed.
+    * Allows customer choose the drink they want, and change refill easily.
+1. Smart Cup: A cup that monitors liquid levels in the cup and automatically alerts the bar/waiter when the cup is below a certain level (adjustable by customer). Would also have a button for general assistance.
+    * Customizable to individual needs, so the thirsty customer could order a refill when their drink is half full, or a slower customer could get their refill when the drink is completely empty (or no refill at all).
```

Comment:
Great point here. Not everyone really needs a refill at half full, although some do. This is a product that gives customers more control over customizing and tailoring their experience (which seems like a positive thing).